packages/api/__tests__/api/private/index.ts

```diff
@@ -3357,7 +3357,6 @@ export const ImageList = gql`
         hasNextPage
         hasPreviousPage
       }
-      totalCount
```

Comment:
what is that?

Reply:
@roosnic1  Most probably it's something related to a task finished but we didn't update API